install/antennas.csv

```diff
@@ -335,7 +336,7 @@ Trimble Navigation Ltd.,TRM57971.00,4127561911,RGHL,0.002,0,0,0,2017-06-08T01:30
 Trimble Navigation Ltd.,TRM57971.00,4127561914,WPAW,0.002,0,0,0,2016-08-15T02:08:08Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM57971.00,4127561917,VGPK,0.055,0,0,0,2016-08-23T01:10:33Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM57971.00,4127561919,TGWH,0.002,0,0,0,2016-10-12T03:04:20Z,9999-01-01T00:00:00Z
-Trimble Navigation Ltd.,TRM57971.00,4127561923,LOK1,0,0,0,0,2016-11-15T21:44:30Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRM57971.00,4127561923,LOK1,0,0,0,0,2016-11-15T21:44:30Z,2017-09-20T03:24:30Z
```

Comment:
this one (and GDS1 at line 348) are a bit weird.
Both sites have their receiver swapped on `2017-09-20`. They're both offline, so we don't know the exact timestamp to start the cgps-session of the latest receiver. Both sites still have the same antenna and, looking at _magma_ web interface, the antenna end dates are "open".